.gitignore

```diff
@@ -39,3 +39,6 @@ github_pages/doc/
 
 # library lock files
 /libs/**/Cargo.lock
+
+# TODO: Remove once std is upstreamed
+.cargo/
```

Comment:
what is this, something that the build of `rust` generates?

Reply:
Yes, we set the specific path to our custom `rustc` installation in `.cargo/config.toml`.